src/main/scala/com/gu/scanamo/Scanamo.scala

```diff
@@ -62,9 +81,28 @@ object Scanamo {
     * None
     * }}}
     */
-  def delete[K, T](client: AmazonDynamoDB)(tableName: String)(key: (Symbol, K)*)
-    (implicit fk: DynamoFormat[K], ft: DynamoFormat[T]): DeleteItemResult =
-    client.deleteItem(deleteRequest(tableName)(key: _*))
+  def delete[HK, T](client: AmazonDynamoDB)(tableName: String)(hashkey: (Symbol, HK))
+    (implicit fhk: DynamoFormat[HK], ft: DynamoFormat[T]): DeleteItemResult =
```

Comment:
@philwills The `T` type param and `ft` implicit param are not necessary, could be removed?